internal/metadata/testdata/qa/valid/valid.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: move2kube.io/v1alpha1
```

Comment:
for consistency just add the `---` and `...` marker like all the other test data files.